src/Main.hs

```diff
@@ -13,19 +11,22 @@ main :: IO ()
 main = execParser opts >>= run
 
 run :: Config -> IO ()
-run (Config w p) = T.putStrLn . wrap w . parseFile =<< contentsOrSTDIN p
+run c = T.putStrLn . wrap c . parseFile =<< contentsOrSTDIN (path c)
 
 contentsOrSTDIN :: FilePath -> IO Text
 contentsOrSTDIN "-" = T.getContents
 contentsOrSTDIN p = T.readFile p
 
-wrap :: Int -> [Content] -> Text
-wrap w = T.stripEnd . T.unlines . concatMap (wrapContent w)
+wrap :: Config -> [Content] -> Text
+wrap c = T.stripEnd . T.unlines . concatMap (wrapContent c)
```

Comment:
Looks like you're using `$` generally, not parentheses. This could also be fully point-free, if you want.